src/main/java/thestonedturtle/mahoganyhomes/TeleportItems.java

```diff
@@ -0,0 +1,186 @@
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE, 71),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE_T, 71)
+	)),
+	TAU(Arrays.asList(
+			new TeleportItem(ItemID.SKILLS_NECKLACE6, 45),
+			new TeleportItem(ItemID.SKILLS_NECKLACE5, 45),
+			new TeleportItem(ItemID.SKILLS_NECKLACE4, 45),
+			new TeleportItem(ItemID.SKILLS_NECKLACE3, 45),
+			new TeleportItem(ItemID.SKILLS_NECKLACE2, 45),
+			new TeleportItem(ItemID.SKILLS_NECKLACE1, 45),
+			new TeleportItem(ItemID.RING_OF_WEALTH_I5, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_I4, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_I3, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_I2, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_I1, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_5, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_4, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_3, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_2, 47),
+			new TeleportItem(ItemID.RING_OF_WEALTH_1, 47),
+			new TeleportItem(ItemID.FALADOR_TELEPORT, 78),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE, 79),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE_T, 79)
+	)),
+
+	// Hosidius
+	BARBARA(Arrays.asList(
+			new TeleportItem(ItemID.TELEPORT_TO_HOUSE, 14),
+			new TeleportItem(ItemID.HOSIDIUS_TELEPORT, 14),
+			new TeleportItem(ItemID.XERICS_TALISMAN, 27),
+			new TeleportItem(ItemID.KHAREDSTS_MEMOIRS, 70),
+			new TeleportItem(ItemID.BOOK_OF_THE_DEAD, 70)
+	)),
+	LEELA(Arrays.asList(
+			new TeleportItem(ItemID.XERICS_TALISMAN, 30),
+			new TeleportItem(ItemID.KHAREDSTS_MEMOIRS, 67),
+			new TeleportItem(ItemID.BOOK_OF_THE_DEAD, 67),
+			new TeleportItem(ItemID.TELEPORT_TO_HOUSE, 72),
+			new TeleportItem(ItemID.HOSIDIUS_TELEPORT, 72)
+	)),
+	MARIAH(Arrays.asList(
+			new TeleportItem(ItemID.XERICS_TALISMAN, 50),
+			new TeleportItem(ItemID.KHAREDSTS_MEMOIRS, 56),
+			new TeleportItem(ItemID.BOOK_OF_THE_DEAD, 56),
+			new TeleportItem(ItemID.TELEPORT_TO_HOUSE, 100),
+			new TeleportItem(ItemID.HOSIDIUS_TELEPORT, 100)
+	)),
+
+	// Varrock
+	BOB(Arrays.asList(
+			new TeleportItem(ItemID.VARROCK_TELEPORT, 60),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE, 100),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE_T, 100)
+	)),
+	JEFF(Arrays.asList(
+			new TeleportItem(ItemID.VARROCK_TELEPORT, 23),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE, 30),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE_T, 30)
+	)),
+	SARAH(Arrays.asList(
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE, 28),
+			new TeleportItem(ItemID.ACHIEVEMENT_DIARY_CAPE_T, 28),
+			new TeleportItem(ItemID.VARROCK_TELEPORT, 37)
+	));
+
+	private final List<TeleportItem> teleportItems;
+
+	public TeleportItem getClosestTeleportItemOnPlayer(Client client)
+	{
+		ItemContainer inventoryContainer = client.getItemContainer(InventoryID.INVENTORY.getId());
+		ItemContainer equipmentContainer = client.getItemContainer(InventoryID.EQUIPMENT.getId());
+
+		Collections.sort(teleportItems);
```

Comment:
This is not needed as the items are already in order by increasing distance. Because this is not needed my previous comment about changing the constructor is applicable